SECURITY.md

```diff
@@ -20,15 +20,14 @@ Please include the requested information listed below (as much as you can provid
 
   * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
   * Full paths of source file(s) related to the manifestation of the issue
-  * The location of the affected source code (tag/branch/commit or direct URL)
```

Comment:
removing this because we aren't open source, so developers can't exactly point to it.